android/engine/src/test/java/org/smartregister/fhircore/engine/util/DateUtilsTest.kt

```diff
@@ -38,6 +39,27 @@ class DateUtilsTest {
     Assert.assertEquals("3-22-2020", finalDate)
   }
 
+  @Test
+  fun `addDays() should return same date when given initialDate and pattern`() {
```

Comment:
Could we change this to something to reflect that zero days were added? e.g.

`addDays should return same date when default 0 days are added`

Reply:
Yeah sure